src/components/dialogs/create-study-dialog/create-study-dialog.js

```diff
@@ -145,9 +146,11 @@ const CreateStudyDialog = ({ open, onClose, providedExistingCase }) => {
                     setValue(FORMATTED_CASE_PARAMETERS, formattedParams, {
                         shouldDirty: true,
                     });
+                    setCaseFormat(result.formatName);
```

Comment:
Why don't you add it with the other fields?
```suggestion
                    setValue(FORMAT_NAME, result.formatName);
```

Reply:
done, i removed the state